### ISSUE TITLE
Repensar a interface

### ISSUE THREAD

**Issue by ludimila** [prefeiturasp/SME-FilaDaCreche#12]
Atualmente a aplicação não ocupa todo o espaço da tela, funciona muito bem em celular mas o efeito não é muito agradável em compuitadores.

![tela fila](https://user-images.githubusercontent.com/1562636/43648887-0ea881b6-9712-11e8-99f4-d247ceff92be.png)

**Comment by bplmp** [comment]
2/3 do nosso tráfego all-time é mobile; eu tenho dúvidas em quanto vale a pena refatorar a interface para desktop.

Manter a mesma lógica para os dois facilita a manutenção.

**Comment by oieusouamiguel** [comment]
Concordo com @ludimila. Acho interessante pensar no desktop sim. Mesmo que 2/3 do tráfego seja mobile, ainda existe uma galerinha que usa desktop.

**Comment by ludimila** [comment]
@bplmp vai manter a lógica, é mais sobre deixar responsivo e ocupar o espaço vazio e cinza, eu lembro da @campagnucci comentando sobre como seria interessante melhorar a interface, por isso criei a issue.

**Comment by JDias** [comment]
Eu não vejo problema na que estar no ar, até pelo que eu entendi, é uma interface provisória.
Você tem bastantes serviços que talvez no futuro, quem sabe, consiga ser agregado em uma só interface?
Como referência do que eu estou dizendo http://felipevieira.com.br/site/porto-alegre-lanca-aplicativo-para-consulta-de-servicos-na-area-da-saude-essa-e-a-primeira-etapa-do-eufacopoa-que-vai-incorporar-funcionalidades-com-servicos-de-outras-areas-nas-proximas-semanas/

**Comment by ludimila** [comment]
@JDias a que está no ar não tem problemas, mas nem por isso é fechada a melhorias, como você mesmo citou é uma interface provisória e gostaríamos de iniciar os passos para a interface definitiva. Alguma sugestão nesse sentido?

**Comment by JDias** [comment]
não @ludimila , não conheço tão bem assim para poder sugerir algo, a única coisa que eu poderia sugerir, foi o link que coloquei do eufacopoa.

acredito que com pwa os apps já eram, mas a ideia de poder ter apenas uma interface para os vários serviços me parece interessante.

no caso do eufacopoa são varias áreas,  mas acho que a sme-sp tem material o suficiente para ter uma abordagem similar, e em um só lugar/interface, poder acessar dados, mapas, filas, bots etc ...

não sei se vai na linha do que vocês estão pensando, mas seria como eu tentaria abordar.

o pessoal de poa começou https://alfa.portoalegre.rs.gov.br/ , o gov federal lançou o servicos.gov.br , então me parece que faz sentido ter algo que unifique o cardápio de serviços da sme.

**Comment by bplmp** [comment]
Ok, eu não tenho nada contra refatorar a interface mobile se for só modificando CSS.

Faz componentes específicos pra desktop eu acho que vai aumentar o custo de manutenção. Se essa for a decisão, eu sugiro implementar um sistema de testes A/B pra avaliar a melhoria de alguma métrica de "conversão".

**Comment by ludimila** [comment]
@bplmp a ideia é essa mesmo ber

**Comment by oieusouamiguel** [comment]
Eu acho que teriamos que refatorar o UX/UI todo. Não está tão clara as informações e qual a intenção do app em si para que pais e responsáveis usem de fato. A interface em si precisa ser melhorada para dar informações mais claras sobre os procedimentos e, por exemplo, não precisar dar scroll em todas as creches que o usuário procurar para que depois haja uma informação do que fazer. Coisas pequenas que podem ser repensadas com simples wireframes, nem precisa de muito.

**Comment by bplmp** [comment]
Todo apoio para repensar a interface! Só acho que o foco deve ser mobile e que devemos minimizar o custo de manutenção que pode surgir ao ter duas lógicas muito diferentes entre mobile e desktop.

Wireframes são uma ótima maneira pra gente começar a pensar como melhorar.

**Comment by bplmp** [comment]
Discussão sobre tela de resultados em https://github.com/prefeiturasp/SME-FilaDaCreche/issues/27
